conditionals.py

```diff
@@ -62,4 +62,11 @@ def restrict_one_entrance_randomizer(_, random_settings):
     for item in erlist:
         setting, off_option = item.split(":")
         if setting != keepon:
-            random_settings[setting] = off_option
\ No newline at end of file
+            random_settings[setting] = off_option
+
+def workaround_th_scz_bug(weight_dict, random_settings):
+    """ Makes sure https://github.com/TestRunnerSRL/OoT-Randomizer/issues/1331 won't happen. """
+    weights = weight_dict['triforce_goal_per_world']
```

Comment:
don't need this line

Reply:
That would leave a use of an undefined variable in line 72.